dev/com.ibm.websphere.javaee.activity.1.0/src/javax/activity/InvalidActivityException.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package javax.activity;
+
+import java.rmi.RemoteException;
+
+/**
+ * A mirror of the corresponding class that was provided in JDK 5-8, and removed in JDK 9+
+ */
+@SuppressWarnings("serial")
```

Comment:
This should not be used on an Exception class, especially one that extends RemoteException..... if the serialversionUID doesn't match, then this class cannot be returned to remote clients.
Really need to find the serialversionUID that is in the classes removed from the JDK to ensure interoperability.

Reply:
It appears the original classes did not provide a serialVersionUID either; nor any instance variables... so I'm approving.  May be nice to have a unit test that serializes using the Java 8 class, then deserializes using this class; but that is a nice to have.